examples/enclave_manager/Makefile

```diff
@@ -15,20 +15,19 @@
 # Set SGX_MODE default (which can be overridden with an environment variable):
 SGX_MODE?=SIM
 
-PY_VERSION=${shell python3 --version | sed 's/Python \(3\.[0-9]\).*/\1/'}
+PY_VERSION=${shell python3 --version | sed 's/Python \(3\.[0-9]\).*/\1/' | tr -d .}
 MOD_VERSION=${shell ../../bin/get_version}
 
-EGG_FILE=dist/tcf_enclave_manager-${MOD_VERSION}-py${PY_VERSION}-linux-x86_64.egg
+WHEEL_FILE=dist/tcf_enclave_manager-${MOD_VERSION}-cp${PY_VERSION}-cp${PY_VERSION}m-linux_x86_64.whl
```

Comment:
We have one more egg file in "examples/common/python/Makefile". Are we planning to replace that with whl file in a separate PR ?

Reply:
Its already replaced in this PR.

Reply:
This change requires an undocumented extra step:
`pip3 install wheel`
This was found from here:  https://github.com/tensorflow/tensorflow/issues/348

The correction to this regression is at #144